### ISSUE TITLE
Classification problem

### ISSUE THREAD

**Issue by KirinHao** [lhoyer/MIC#78]
I have some problems with the data set used in the classification section. 
In the classification experiment, is only the training set and validation set of the VisDA-2017 data set used? Is the test set directly VisDA-2017 validation set? I know little about this dataset and look forward to your reply.
Thank you very much!
```
    # in cls.vision.datasets.util.py
    elif dataset_name in datasets.__dict__:
        # load datasets from common.vision.datasets
        dataset = datasets.__dict__[dataset_name]

        def concat_dataset(tasks, **kwargs):
            return ConcatDataset([dataset(task=task, **kwargs) for task in tasks])

        train_source_dataset = concat_dataset(root=root, tasks=source, download=True, transform=train_source_transform)
        train_target_dataset = concat_dataset(root=root, tasks=target, download=True, transform=train_target_transform)
        val_dataset = concat_dataset(root=root, tasks=target, download=True, transform=val_transform)
        if dataset_name == 'DomainNet':
            test_dataset = concat_dataset(root=root, tasks=target, split='test', download=True, transform=val_transform)
        else:
            test_dataset = val_dataset  # about here
```

**Comment by KirinHao** [comment]
In addition, my questions include that this is also used in data set files (only training sets and validation sets are used). Look forward to your reply very much, thank you!
```
    # in cls.vision.datasets.visda2017.py
    download_list = [
        ("image_list", "image_list.zip", "https://cloud.tsinghua.edu.cn/f/c107de37b8094c5398dc/?dl=1"),
        ("train", "train.tgz", "https://cloud.tsinghua.edu.cn/f/c5f3ce59139144ec8221/?dl=1"),
        ("validation", "validation.tgz", "https://cloud.tsinghua.edu.cn/f/da70e4b1cf514ecea562/?dl=1")
    ]
    image_list = {
        "Synthetic": "image_list/train.txt",
        "Real": "image_list/validation.txt"
    }
    CLASSES = ['aeroplane', 'bicycle', 'bus', 'car', 'horse', 'knife',
               'motorcycle', 'person', 'plant', 'skateboard', 'train', 'truck']
```
